spring-native/storage-blob/README.md

```diff
@@ -0,0 +1,255 @@
+
+#### Run with Bash
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=./terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=./terraform apply -auto-approve
+
+```
+
+#### Run with Native Tools Command Prompt
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=terraform apply -auto-approve
+
+```
+
+It may take a few minutes to run the script. After successful running, you will see prompt information like below:
+
+```shell
+
+azurerm_resource_group.main: Creating...
+azurerm_resource_group.main: Creation complete after 3s ...
+azurerm_storage_account.application: Creating...
+azurerm_storage_account.application: Still creating... [10s elapsed]
+azurerm_storage_account.application: Creation complete after 39s ...
+azurerm_storage_container.application: Creating...
+azurerm_storage_container.application: Creation complete after 1s ...
+azurerm_role_assignment.storage_blob_contributor: Creating...
+azurerm_role_assignment.storage_blob_contributor: Still creating... [20s elapsed]
+azurerm_role_assignment.storage_blob_contributor: Creation complete after 26s...
+
+...
+Apply complete! Resources: 6 added, 0 changed, 0 destroyed.
+
+```
+
+You can go to [Azure portal](https://ms.portal.azure.com/) in your web browser to check the resources you created.
+
+### Export Output to Your Local Environment
+
+NOTE: If you are building a lightweight container containing a native executable, you can skip this section step.
+
+Running the command below to export environment values:
+
+#### Run with Bash
+
+```shell
+source ./terraform/setup_env.sh
+```
+
+#### Run with Native Tools Command Prompt
+
+```shell
+terraform\setup_env.bat
+```
+
+If you want to run the sample in debug mode, you can save the output value.
+
+```shell
+AZURE_STORAGE_ACCOUNT=...
+```
+
+## Run Locally
+
+### Run the sample with Maven
+
+In your terminal, run `mvn clean spring-boot:run`.
+
+```shell
+mvn clean spring-boot:run
+```
+
+### Run the sample in IDEs
+
+You can debug your sample by adding the saved output values to the tool's environment variables or the sample's `application.yaml` file.
+
+* If your tool is `IDEA`, please refer to [Debug your first Java application](https://www.jetbrains.com/help/idea/debugging-your-first-java-application.html) and [add environment variables](https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html#add-environment-variables).
+
+* If your tool is `ECLIPSE`, please refer to [Debugging the Eclipse IDE for Java Developers](https://www.eclipse.org/community/eclipse_newsletter/2017/june/article1.php) and [Eclipse Environment Variable Setup](https://examples.javacodegeeks.com/desktop-java/ide/eclipse/eclipse-environment-variable-setup-example/).
```

Comment:
Maybe we can remove this two here, just to demo how to run as native?

Reply:
OK